src/model/mosaic/MosaicId.ts

```diff
@@ -33,6 +41,31 @@ export class MosaicId {
      */
     public readonly fullName?: string;
 
+    /**
+     * Create a random MosaicId for given `owner` PublicAccount.
+     * 
+     * @param   owner   {PublicAccount}
+     * @return  {MosaicId}
+     */
+    public static createRandom(owner: PublicAccount): MosaicId {
```

Comment:
Remove ``createRandom(owner: PublicAccount)`` from mosaicId. The nonce can't be retrieved from the MosaicDefinitionTransaction.

Reply:
Fixed with: https://github.com/nemtech/nem2-sdk-typescript-javascript/pull/68/commits/24fba0b30f6f98b34d3e02477e092cc20ecd33b9